azure/azure-release-pipeline.yml

```diff
@@ -68,7 +68,5 @@ extends:
       short_service_name: "${{ variables.short_service_name }}-sa"
       product_display_name: "${{ variables.product_display_name }} (ERSSA)"
       service_base_path: "${{ variables.service_base_path }}-sa"
-      jinja_templates:
-        ERS_TARGET_SERVER: e-referrals-service-api--sa
```

Comment:
Is this no longer required? How do we now define the proxy name for each environment?

Reply:
It is no longer required after https://github.com/NHSDigital/api-management-infrastructure/pull/302/files has been merged and deployed.

It was wrong to have an explicitly named target server for SA (it should be just e-referrals-service-api) for any Apigee environment that only has one e-RS environment mapped.

Another way to put it: only Apigee's "internal-dev" has >1 e-RS environments where we need to distinguish between different target servers.